parm/stage/stage.yaml.j2

```diff
@@ -0,0 +1,204 @@
+#############################################################
```

Comment:
Not sure why we have a singular `stage.yaml.j2` for both GFS and GEFS.
There would be a `stage_gfs_default.yaml.j2` and `stage_gefs_default.yaml.j2`
The `config/gfs|gefs/stage_ic.sh` could point to the default ones for their respective yamls.

Reply:
The current staging job only differentiated GFS/GEFS based on what the starting member would be:
https://github.com/NOAA-EMC/global-workflow/blob/develop/scripts/exglobal_stage_ic.sh#L15

Otherwise, they overlap and allow for a single yaml. A gefs version of the stage yaml would be a copy paste from the current one for all of the sections it needs.

If you would prefer separate yamls based on GFS vs GEFS, then I can do that. Let me know.

Reply:
I think having separate yamls makes sense, otherwise the complexity to fold everything in a single yaml becomes unmanageable. 

At setup_expt.py time, we know whether its `gfs|gefs`, and we also know whether it is `forecast-only|cycled`, it shouldn't be that difficult to put in the appropriate `stage.yaml.j2` into `STAGE_IC_YAML_TMPL`.

Reply:
Have separated the main yaml into a master ones for gfs and gefs, and also broke the contents out into component yamls. The master ones handle cycle/mem loop setup and then do includes of the component yamls. Have tested successfully on Hera.